### ISSUE TITLE
"make" run fails

### ISSUE THREAD

**Issue by djs55** [mirage/mirage-www#337]
```
$ make run
cd src && sudo make run
Password:
make: *** No rule to make target `run'.  Stop.
make: *** [run] Error 2
```
Possibly this target should be removed?

**Comment by mor1** [comment]
As a general rule, `run` targets should be removed from all the various
mirage Makefiles I think -- never worked reliably, even less likely to work
now...

On 17 June 2015 at 14:18, Dave Scott <notifications@github.com> wrote:

> $ make run
> cd src && sudo make run
> Password:
> make: *** No rule to make target `run'.  Stop.
> make: *** [run] Error 2
>
> Possibly this target should be removed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mirage/mirage-www/issues/337>.
>



-- 
Richard Mortier
mort@cantab.net

**Comment by samoht** [comment]
yes, we should remove all the mention to `make run`. The base Makefile is broken, I usually just go into `src/` and configure here directly. PR are welcome to fix this.

**Comment by dongcarl** [comment]
fixed the unix bits of hello-world in #364

**Comment by mor1** [comment]
Closing as all mention of `make run` should now have been removed from the intro pages (hello-world, building the website, deployment). Are there others?
